tools/gui/qtkontakt.py

```diff
@@ -257,12 +262,7 @@ def __check_werte(self, kontaktdaten: dict):
         if self.modus == Modus.TERMIN_SUCHEN:
             kontakt_tools.validate_kontaktdaten(kontaktdaten)
         elif self.modus == Modus.CODE_GENERIEREN:
-            kontakt_tools.validate_plz_impfzentren(kontaktdaten["plz_impfzentren"])
-            kontakt_tools.validate_email(kontaktdaten["kontakt"]["notificationReceiver"])
-            try:
-                kontakt_tools.validate_phone(kontaktdaten["kontakt"]["phone"])
-            except ValidationError as error:
-                raise ValidationError("Telefonnummer: +49 nicht vergessen") from error
+            kontakt_tools.validate_kontaktdaten(kontaktdaten)
```

Comment:
If-Abfrage ist hier dann unnötig geworden
```suggestion
        kontakt_tools.validate_kontaktdaten(kontaktdaten)
```